packages/network/src/createTxQueue.ts

```diff
@@ -25,9 +25,10 @@ export function createTxQueue<C extends Contracts>(
   computedContracts: IComputedValue<C> | IObservableValue<C>,
   network: Network,
   gasPrice$: BehaviorSubject<number>,
-  options?: { concurrency?: number; devMode?: boolean }
+  options?: { concurrency?: number; devMode?: boolean; maxRetries?: number }
 ): { txQueue: TxQueue<C>; dispose: () => void; ready: IComputedValue<boolean | undefined> } {
   const { concurrency } = options || {};
+  const maxRetries = options?.maxRetries ?? 3;
```

Comment:
should we make `0` the default for now to not unexpectedly change then behaviour for other consumers?

Reply:
i kind of want to change the default behavior for people to add to more of the magic of MUD. do you see a reason why it would be bad or just don't want to surprise people?

Reply:
3 is a little aggressive though... 1 probably makes more sense as a default

Reply:
Mainly to avoid surprising people, in general I think it's a good default behaviour, so also happy with enabling by default. Agree 1 is a better default than 3 though. We should also add a note about this behaviour in the comment above the function, so it can get parsed into the documentation here https://mud.dev/packages/network/api/#createtxqueue